src/main.py

```diff
@@ -40,7 +43,18 @@ def main() -> None:
     """
 
     while True:
-        get_input()
+        #get imput 
+        for event in pg.event.get():
+            if event.type == pg.KEYDOWN:
+                if event.key == pg.K_UP or pg.K_SPACE:
```

Comment:
To keep the code modular it  would be better to add a list of jump_keys and a list of sneak_kes and ask if selected key is in that list like:
```suggestion
        jump_keys: list[pg.Key] = [pg.K_UP, pg.K_SPACE]
                                if event.key in jump_keys:
```